packages/react-devtools-shared/src/Logger.js

```diff
@@ -31,6 +31,18 @@ export type LogEvent =
       +duration_ms: number,
       +inspected_element_display_name: string | null,
       +inspected_element_number_of_hooks: number | null,
+    |}
+  | {|
+      +event_name: 'select-element-in-tree',
+    |}
+  | {|
+      +event_name: 'select-element-in-owner-view',
+    |}
+  | {|
+      +event_name: 'inspect-native-element',
+    |}
```

Comment:
Besides the event, do we care about which elements were selected?

Reply:
Yes, I want to discuss with you about adding a "metadata" (maybe not the best name) column to DB. It can be useful for several usage logging (not included in this PR by purpose). But for this one, how do you imagine the additional info about the elements be useful for us?